staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager_test.go

```diff
@@ -139,15 +151,15 @@ func (f *TestFieldManager) Reset() {
 }
 
 func (f *TestFieldManager) Apply(obj runtime.Object, manager string, force bool) error {
-	out, err := fieldmanager.NewFieldManager(f.fieldManager).Apply(f.liveObj, obj, manager, force)
+	out, err := fieldmanager.NewFieldManager(f.fieldManager, f.typeConverter, f.objectConverter, f.groupVersion).Apply(f.liveObj, obj, manager, f.userAgent, force)
```

Comment:
Do you know why we don't create the `FieldManager` in f? it doesn't seem like we need to create it here, especially now that it's more complicated.

Reply:
I'll try this

Reply:
Done.